.github/workflows/build.yml

```
@@ -11,7 +11,7 @@ jobs:
     needs: ['cpp-lint', 'js-lint']
     strategy:
       matrix:
-        version: [14, 15, 16, 17, 18, 19, 20, 21]
+        version: [14, 15, 16, 17, 18, 19, 20, 21, 22]
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
```

and upgrade `actions/checkout` :P?
```suggestion
      - uses: actions/checkout@v4
```

---

https://github.com/actions/checkout/releases/tag/v4.1.4

---

We could, but I don't see that necessary for this PR. Any reason to update it?